Cargo.toml

```diff
@@ -39,6 +39,7 @@ opentelemetry_sdk = { version = "0.28.0", features = ["rt-tokio"] }
 tracing-opentelemetry = "0.29.0"
 futures = "0.3.31"
 pin-project = "1.1.10"
+dotenv = "0.15.0"
```

Comment:
`dotenv`[is unmaintained, ](https://rustsec.org/advisories/RUSTSEC-2021-0141.html), lets use [dotenvy](https://crates.io/crates/dotenvy) instead.